src/kirby/components/toolbar/toolbar.component.tns.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  ContentChild,
+  TemplateRef,
+  Output,
+  EventEmitter,
+} from '@angular/core';
+import { isAndroid } from 'tns-core-modules/platform';
+
+import { ToolbarEndElementDirective, ToolbarSecondaryElementDirective } from './toolbar.directive';
+
+@Component({
+  selector: 'kirby-toolbar',
+  templateUrl: './toolbar.component.html',
+  styleUrls: ['./toolbar.component.scss'],
+})
+export class ToolbarComponent implements OnInit {
+  @Input() title: string;
+  @Input() showBackButton: boolean;
```

Comment:
i think 99% of the time you actually want to show the back button. So was it may an idea to revert this in to `hideBackButton`?